pkgs/games/tome2/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub,  makeDesktopItem, ncurses, libX11, boost, cmake }:
+
+let
+  pname = "tome2";
+  desc = "A dungeon crawler similar to Angband, based on the works of Tolkien.";
+
+  desktop = makeDesktopItem {
+    desktopName = pname;
+    name = pname;
+    exec = "@out@/bin/${pname}-x11";
+    icon = "${pname}";
+    terminal = "False";
+    comment = desc;
+    type = "Application";
+    categories = "Game;RolePlaying;";
+    genericName = pname;
+  };
+
+in stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  version = "2.4";
+
+  src = fetchFromGitHub {
+        owner = "tome2";
+        repo = "tome2";
+        rev = "4e6a906c80ff07b75a6acf4ff585b47303805e46";
+        sha256 = "06bddj55y673d7bnzblk8n01z32l6k2rad3bpzr8dmw464hx4wwf";
+      };
```

Comment:
2 space indent to match the rest of the file and the coding standards, and put the closing `};` at the same indentation level as `src`.